test/e2e/node/node_problem_detector.go

```diff
@@ -147,11 +147,11 @@ var _ = SIGDescribe("NodeProblemDetector [DisabledForLargeClusters]", func() {
 		for i := 1; i <= numIterations; i++ {
 			for j, host := range hosts {
 				if isStandaloneMode[host] {
-					rss, workingSet := getMemoryStat(f, host)
+					rss, workingSet := getMemoryStat(host)
 					rssStats[host] = append(rssStats[host], rss)
 					workingSetStats[host] = append(workingSetStats[host], workingSet)
 					if i == numIterations {
-						cpuUsage, uptime := getCPUStat(f, host)
+						cpuUsage, uptime := getCPUStat(host)
 						cpuUsageStats[host] = append(cpuUsageStats[host], cpuUsage)
 						uptimeStats[host] = append(uptimeStats[host], uptime)
```

Comment:
I think the overall idea is good but there might be several references in the code where the parameters that are being proposed to be deleted might be required. More details in the conversation.